lib/jekyll/assets/liquid/tag/proxied_asset.rb

```diff
@@ -77,6 +79,16 @@ def digest
           end
 
           # ------------------------------------------------------------------
+
+          def content_type=(type)
+            return if @content_type == type
+
+            path = Pathutil.new asset.logical_path
```

Comment:
Methods that have arguments should have `()` for clarity.  Unless it's a non-sub-local `asset arg` would be fine (I would let it slip) but `asset.blah val` will probably end up getting corrected later so it's clearer.